.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
can we remove this file?
add `.vscode` to `.gitignore`

Reply:
Sure, I just put this file for run the tests with F5.

Reply:
I just removed the file and sent another commit. Tks.